src/app/src/main/kotlin/org/imperial/mrc/hint/exceptions/ExceptionHandler.kt

```diff
@@ -24,4 +27,16 @@ class HintExceptionHandler : ResponseEntityExceptionHandler() {
                                          request: WebRequest): ResponseEntity<Any> {
         return ErrorDetail(status, e.message ?: "Something went wrong").toResponseEntity()
     }
+
+    @ExceptionHandler(HintException::class)
+    protected fun handleHintException(e: HintException): ResponseEntity<Any>
+    {
+        return ErrorDetail(e.httpStatus, e.message!!).toResponseEntity()
+    }
+
+    @ExceptionHandler(ConstraintViolationException::class)
```

Comment:
This is needed to handle the exception thrown when the controller method parameter validation fails